units.yaml

```diff
@@ -10196,7 +10196,7 @@
     - id: "mi/gal"
       ascii: "mi/gal"
       html: "mi/gal"
-      mathml: "<mrow><mi mathvariant='normal'>mi</mi><mo>/</mo><mrow><mi mathvariant='normal'>gal</mi></mrow>"
+      mathml: "<mrow><mi mathvariant='normal'>mi</mi><mo>/</mo><mrow><mi mathvariant='normal'>gal</mi></mrow></mrow>"
```

Comment:
This is wrong. The first and second `mrow` should not be embedded in each other.

Reply:
You want this instead:
```xml
<mrow><mi mathvariant='normal'>mi</mi></mrow><mo>/</mo><mrow><mi mathvariant='normal'>gal</mi></mrow>
```